samples/README.md

```diff
@@ -37,19 +37,21 @@ These policies provide additional best practices and are worthy of close conside
 1. [Restrict User Group](CheckUserGroup.md)
 1. [Require pods are labeled](RequireLabels.md)
 1. [Require pods have certain labels](RequireCertainLabels.md)
+1. [Require Deployments have multiple replicas](RequireDeploymentsHaveReplicas.md)
 
 ## Applying the sample policies
 
 To apply these policies to your cluster, install Kyverno and import the policies as follows:
 
-**Install Kyverno**
+### Install Kyverno**
 
 ````sh
 kubectl create -f https://github.com/kyverno/kyverno/raw/master/definitions/install.yaml
```

Comment:
As Kyverno introduced the release folder to hold the official released manifest, can you please update the install command to: 
```
kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml
```
Thanks!

Reply:
resolved with latest commit